NYPL/cantaloupe
# DR-3003 first stab at serving tiffs

## JIRA ticket:
- [Enable access to TIF file download through IIIF](https://newyorkpubliclibrary.atlassian.net/browse/DR-3003)

## Things Done:
- Allowed the shim to recognize the `u` derivative type

## How to test:
- Rebuild your cantaloupe containers (but first do the thing from the README in `nginx-configs/image_server_to_iiif.js` for local development)
- Get on the VPN
- Run the app
- Visit http://localhost:8080/index.php?id=53926&t=u. Your browser should download the .tif file corresponding to 'A Plan of the City of New York'
- Visit http://localhost:8182/iiif/2/53926/full/full/0/default.tif. The same behavior should occur.
- I guess we'll have to see how this works with larger images (such as https://link.nypl.org/2bpdoSuDQmqJlWyx-f_m_Ar)
